Apps/W1/Shopify/app/src/Products/Codeunits/ShpfyCreateProduct.Codeunit.al

```diff
@@ -247,7 +175,7 @@ codeunit 30174 "Shpfy Create Product"
     /// <param name="VariantCode">Parameter of type Code[10].</param>
     /// <param name="UoM">Parameter of type Code[10].</param>
     /// <returns>Return value of type Text.</returns>
```

Comment:
XML Summary is outdated after the change of the return type

Reply:
Updated.